tests/mlmodel_langchain/test_vectorstore.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import langchain
+from langchain.document_loaders import PyPDFLoader
+from langchain.embeddings.openai import OpenAIEmbeddings
+from langchain.vectorstores import FAISS
+from testing_support.fixtures import (
+    reset_core_stats_engine,
+    validate_custom_event_count,
+)
+from testing_support.validators.validate_custom_events import validate_custom_events
+from testing_support.validators.validate_transaction_metrics import (
+    validate_transaction_metrics,
+)
+
+from newrelic.api.background_task import background_task
+
+vectorstore_recorded_events = [
+    (
+        {"type": "LlmEmbedding", "timestamp": 1702052394446},
```

Comment:
We don't need to validate the openai llm events here, only the one's for langchain instrumentation. Otherwise we'll end up having to modify test code in multiple test areas which will get confusing. We can just add a note to the event count to say 2 of the events are generated by openai embedding instrumentation.